pkg/cli/common.go

```diff
@@ -92,6 +92,9 @@ type BudResults struct {
 	LogRusage           bool
 	RusageLogFile       string
 	UnsetEnvs           []string
+	OSVersion           string
+	OSFeatures          []string
+	Envs                []string
```

Comment:
There seems to be something off with tab/spaces with this in comparison to the `Jobs` and above line.  Can you at least alphabetize the three that you've added in the upper part of this struct please?  Well as much a possible, I see other rogue entries there.